internal/custom-code-logging/command_custom_code_logging_update.go

```diff
@@ -21,14 +21,18 @@ func (c *customCodeLoggingCommand) newUpdateCommand() *cobra.Command {
 		Args:  cobra.ExactArgs(1),
 		RunE:  c.update,
 	}
-
+	pcmd.AddEnvironmentFlag(cmd, c.AuthenticatedCLICommand)
 	cmd.Flags().String("log-level", "INFO", fmt.Sprintf("Specify the Custom Code Logging Log Level as %s.", utils.ArrayToCommaDelimitedString(allowedLogLevels, "or")))
 	cmd.MarkFlagsOneRequired("log-level")
 	pcmd.AddContextFlag(cmd, c.CLICommand)
 	return cmd
 }
 
 func (c *customCodeLoggingCommand) update(cmd *cobra.Command, args []string) error {
+	environment, err := c.Context.EnvironmentId()
```

Comment:
Same issue here as `list`

Reply:
<img width="781" alt="Screenshot 2024-11-14 at 3 50 18 PM" src="https://github.com/user-attachments/assets/9f60d34d-3c69-4563-a90c-0eb486c5d0c6">
@sgagniere said `This function will return the environment id from the flag if they used it, otherwise it returns the current active environment id.`